src/python/install.sh

```diff
@@ -454,6 +454,16 @@ if [[ "${INSTALL_PYTHON_TOOLS}" = "true" ]] && [[ $(python --version) != "" ]];
             echo "${util} already installed. Skipping."
         fi
     done
+    
```

Comment:
Can you bump the `patch` version for this Feature? (update it to `1.3.2`)

https://github.com/devcontainers/features/blob/aee4d8ae950209eb4aeacc4d7f8781c3bacf4dd7/src/python/devcontainer-feature.json#L3

You can read more about versioning in https://containers.dev/implementors/features-distribution/#versioning

Reply:
ok done